roscala/src/main/scala/coop/rchain/rosette/Tuple.scala

```diff
@@ -12,6 +12,12 @@ case object AbsentRest extends TupleError
 case object InvalidRest extends TupleError
 
 case class Tuple(elem: Seq[Ob]) extends Ob {
+  //TODO revisit
+  val metaTup: Ob = Ob.INVALID
+  val sboTup: Ob = Ob.INVALID
```

Comment:
What does sbo mean?

Reply:
Quick explanation why you introduced this also would be nice :)

Reply:
SBO means `shared behavior object`, according to Rosette reference. This was introduced because `meta` and `parent` for the tuple should be `CLASS_META(Tuple)` and `CLASS_SBO(Tuple)` respectively. I just skipped the implementation for now.

Reply:
I'm not sure if we will ever need this for `Tuple`. Let's remove it for now and add it again in case we actually need it.

Reply:
In the C++ version, the `elem` is a `slot`[2:] slice, so it may be necessary to "mock" first two slots, but it is OK for me to remove this for now.